packages/graphql-mocks/src/pack/pack.ts

```diff
@@ -9,7 +7,7 @@ export const pack: Packer = async function pack(
   middlewares = [],
   packOptions = defaultPackOptions,
 ) {
-  middlewares = [...middlewares, embed({ wrappers: [embedPackOptionsWrapper] })];
```

Comment:
`embedPackOptionsWrapper` isn't a great way to ensure that `pack` is added to context and not tampered with. It can be added before the query is called and frozen so it isn't tampered with.

Reply:
Done in #275